airflow/dags/ingestion/ask-astro-load-airflow-docs.py

```diff
@@ -21,6 +23,12 @@
 schedule_interval = "0 5 * * *" if ask_astro_env == "prod" else None
 
 
+@task
+def split_docs(urls, chunk_size=100) -> list[list[pd.Dataframe]]:
```

Comment:
I feel split docs as a separate task looks better for user

Reply:
what do you mean by "split docs as a separate"? This is a task.

Reply:
I think this DAG just shows two tasks extract and ingest. There should also be split

Reply:
No, it should be 3
<img width="1482" alt="Screenshot 2024-01-11 at 3 34 50 PM" src="https://github.com/astronomer/ask-astro/assets/98807258/e1b18e78-fc8a-476e-a228-fd86f2790268">